src/wp-includes/class-wp-duotone.php

```diff
@@ -209,10 +209,7 @@ private static function colord_parse_hue( $value, $unit = 'deg' ) {
 			'rad'  => 360 / ( M_PI * 2 ),
 		);
 
-		$factor = $angle_units[ $unit ];
-		if ( ! $factor ) {
-			$factor = 1;
-		}
+		$factor = $angle_units[ $unit ] ?? 1;
```

Comment:
The `??` operator usage decision is still pending. Please refer to the discussion here: [link](https://wordpress.slack.com/archives/C02RQBWTW/p1706780040940549?thread_ts=1706779467.750519&cid=C02RQBWTW).

Reply:
I thought it was accepted :grimacing:. Updated it.